pull-request-practice.py

```diff
@@ -0,0 +1,6 @@
+def review():
```

Comment:
```suggestion
def review() -> None:
```
戻り値がないときにも型ヒントをつけてください。